azure-functions/src/send_grid/message_builder.rs

```diff
@@ -19,8 +19,10 @@ impl MessageBuilder {
     /// # Examples
     ///
     /// ```rust
-    /// # use azure_functions::send_grid::MessageBuilder;
-    /// let message = MessageBuilder::new().to("foo@example.com").build();
+    /// # use azure_functions::bindings::SendGridMessage;
```

Comment:
I probably should have commented on this earlier.

Since this is the documentation on `MessageBuilder` itself, I think we should leave as-is, but change the `build` call to `finish`.  It'd be a little weird to view the documentation for `MessageBuilder` and see examples using `SendGridMessage`.

The same goes for the other doc comment changes that use `SendGridMessage`.

Reply:
Will do. Asked a question above about doing the same in `response_builder`.

Reply:
Hah, not very consistent was I? Let's update the `ResponseBuilder` docs to use `ResponseBuilder` directly.  I think the docs should go with the type / methods being documented, even if we expect users to instantiate a builder probably from the binding types.

Reply:
Agreed